packages/ecommerce-app-base/src/Editor/SortableList.tsx

```diff
@@ -9,25 +9,23 @@ export interface Props {
   skuType?: string;
 }
 
-export const SortableList = SortableContainer<Props>(
-  ({ disabled, deleteFn, productPreviews, skuType }: Props) => {
-    const itemsAreSortable = productPreviews.length > 1;
-    return (
-      <div>
-        {productPreviews.map((product, index) => {
-          return (
-            <SortableListItem
-              disabled={disabled}
-              key={`${product.image}-${product.sku}`}
-              product={product}
-              index={index}
-              onDelete={() => deleteFn(index)}
-              isSortable={itemsAreSortable}
-              skuType={skuType}
-            />
-          );
-        })}
-      </div>
-    );
-  }
-);
+export const SortableList: FC<Props> = ({ disabled, deleteFn, productPreviews = [], skuType }) => {
+  const itemsAreSortable = productPreviews.length > 1;
+
+  return (
+    <>
```

Comment:
- [ ] investigate loss of padding

Reply:
Not sure what happened with the loss of padding, might have been related to some custom css `react-sortable-hoc` was adding.  But I resolved this by adding some padding/margins back as closely as I could using `tokens.spacing.S`

```js
// src/Editor/SortableListItem.tsx
marginTop: tokens.spacingS
```